CONTRIBUTING.md

```diff
@@ -29,25 +29,33 @@ request with instructions on signing the Contributor License Agreement (CLA) if
 you haven't done so. Please follow the instructions to sign the CLA. A `cla:yes`
 tag is then added to the pull request.
 
+![Tag added](https://i.imgur.com/LHEdIfL.png)
+
 ### Step 5. Continous Integration tests
 
-A set of automated tests will also start to run after creating the pull request.
+A set of automated tests will also start to run after creating the pull request
```

Comment:
These tests won't run automatically if the PR author is not in the contributor group. The reviewer need to apply a label with "kokoro force-run" to trigger the build.

Reply:
I put the CI tests step into the code review step since this may happen during the code review.